zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/Job.java

```diff
@@ -70,6 +70,14 @@ public boolean isPending() {
     public boolean isCompleted() {
       return this == FINISHED || this == ERROR || this == ABORT;
     }
+
```

Comment:
I found these methods very useful when debugging the problem, but they are no longer in use.